testing/perl-db_file-lock/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Automatically generated by apkbuild-cpan, template 2
+# Contributor: Timothy Legge <timlegge@gmail.com>
+# Maintainer: Timothy Legge <timlegge@gmail.com>
+pkgname=perl-db_file-lock
+_pkgreal=DB_File-Lock
+pkgver=0.05
+pkgrel=0
+pkgdesc="unknown"
+url="https://metacpan.org/release/DB_File-Lock/"
+arch="noarch"
+license="GPL PerlArtistic"
+cpandepends=""
+cpanmakedepends=""
+cpancheckdepends=""
+depends="$cpandepends"
+makedepends="perl-dev perl-db_file $cpanmakedepends"
+checkdepends="$cpancheckdepends"
+subpackages="$pkgname-doc"
+source="https://cpan.metacpan.org/authors/id/D/DH/DHARRIS/DB_File-Lock-0.05.tar.gz"
+builddir="$srcdir/$_pkgreal-$pkgver"
+
+prepare() {
+	default_prepare
+
+	cd "$builddir"
+	patch -p1 < ../../$pkgname.diff 
```

Comment:
if you rename `$pkgname.diff` to `$pkgname.patch` and add `$pkgname.patch` to `source`, then will default_prepare apply the patch for you.

Reply:
@ncopa is this a matter of an additional line 

source="https://cpan.metacpan.org/authors/id/D/DH/DHARRIS/DB_File-Lock-0.05.tar.gz"
source=$pkgname."patch"

or appended to the current:

source="https://cpan.metacpan.org/authors/id/D/DH/DHARRIS/DB_File-Lock-0.05.tar.gz" "$pkgname.patch"